TerrorConsole/Assets/_Main/Scripts/Shadow/Patrol.cs

```diff
@@ -0,0 +1,43 @@
+using UnityEngine;
+using UnityEngine.AI;
+
+namespace TerrorConsole
+{
+    public class Patrol : MonoBehaviour
+    {
+        public float waitTime;
+        public float startWaitTime;
```

Comment:
We only use `public` for specific cases realted to Data Structures, use `[SerializeField] private`

Reply:
Also the format is `_` before the field name.